src/swap/exolix.ts

```diff
@@ -138,40 +141,60 @@ export function makeExolixPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
             request.toCurrencyCode
           )
 
+    const quoteAmount = parseFloat(exchangeQuoteAmount)
+
     const {
       fromCurrencyCode,
       toCurrencyCode,
       fromMainnetCode,
       toMainnetCode
     } = getCodes(request)
 
-    // Swap the currencies if we need a reverse quote:
-    const quoteParams =
-        request.quoteFor === 'from'
-          ? {
-              coinFrom: fromCurrencyCode,
-              coinFromNetwork: fromMainnetCode,
-              coinTo: toCurrencyCode,
-              coinToNetwork: toMainnetCode,
-              amount: quoteAmount,
-              rateType: 'fixed'
-            }
-          : {
-              coinFrom: toCurrencyCode,
-              coinFromNetwork: toMainnetCode,
-              coinTo: fromCurrencyCode,
-              coinToNetwork: fromMainnetCode,
-              amount: quoteAmount,
-              rateType: 'fixed'
-            }
+    const quoteParams: Record<string, any> = {
+      coinFrom: fromCurrencyCode,
+      coinFromNetwork: fromMainnetCode,
+      coinTo: toCurrencyCode,
+      coinToNetwork: toMainnetCode,
+      amount: quoteAmount,
+      rateType: 'fixed'
+    }
+
+    // Set the withdrawal amount if we are quoting for the toCurrencyCode
+    if (request.quoteFor === 'to') {
+      quoteParams.withdrawalAmount = quoteAmount
```

Comment:
We leave the same `quoteAmount` in `amount` for a `to` quote?

Reply:
Yes, `withdrawalAmount` has a priority over `amount` and we can leave it.